### PR TITLE
updated ECHAM5 metadata

### PR DESCRIPTION
**Pull request by jordansread**
@lawinslow take a quick look? I don't have it pulling out the ranges of the attributes from the actual data, but we are close on the rest of it.

### CODE REVIEW

File: gh_release/ECHAM5_model.xml [hunk 1]
```diff
@@ -158,43 +158,1051 @@
...
+          <rdom>
+            <rdommin>NA</rdommin>
+            <rdommax>NA</rdommax>
+            <attrunit>days before June 1 (in spring only)</attrunit>
+          </rdom>
+        </attrdomv>
+      </attr>
+      <attr>
+        <attrlabl>mean_surf_jul</attrlabl>
+        <attrdef>summer warmth index </attrdef>
```
Comment (lawinslow) [inline gh_release/ECHAM5_model.xml]: Should the attrdef's maybe be the longer-form description in that file? These versions are pretty terse.

Comment (jordansread) [inline gh_release/ECHAM5_model.xml]: Am just using the descriptions in that table, is there a longer-form version than that?

Comment (lawinslow) [inline gh_release/ECHAM5_model.xml]: I was thinking "output details" column. Not "specific metric" column.

Comment (jordansread) [inline gh_release/ECHAM5_model.xml]: oh - I see that Gretchen's paper has a much longer-form version of this

**Pull request merged**